Makefile

```diff
@@ -13,29 +13,43 @@ create-env:
 
 .PHONY: install
 install:
-	$(PIP) install --upgrade $(PIP)
+	$(PIP) install --upgrade pip
+	$(PIP) install .
 	$(PIP) install -r $(TEST_DIR)/utils/requirements.txt
+	$(PIP) install pre-commit pytest pytest-cov codecov
 
 .PHONY: test
 test: .cleanCoverage
-	$(PYTEST) $(TEST_DIR)/unit
+	$(PYTEST) $(TEST_DIR)/unit -v
 
 .cleanCoverage:
 	@echo 'cleaning coverage files ...'
-	rm -f .coverage
-	rm -fr htmlcov/
+	rm -rf .coverage htmlcov/
 
 .PHONY: open-report
 open-report: .cleanCoverage
 	$(PYTEST) --cov=$(YARPC) --cov-report=term-missing --cov-report=html
 	open htmlcov/index.html
 
 .PHONY: ci-test
-ci-test:
-	$(PIP) install -r tests/utils/requirements.txt
-	pre-commit run --all-files
-	$(PIP) install .
-	$(PYTEST) --cov --cov-report=xml
+ci-test: lint
+	$(PYTEST) --cov --cov-report=xml -v
+
+.PHONY: flake8
+flake8:
+	@find . -type d -name 'env' -prune -o -name 'docs' -prune -o -name '*.py' -exec flake8 "{}" +
```

Comment:
Can you explain why do you use `find` here? Why not just `flake8 yarpc` / `flake8 $(SOURSES)` where `SOURCES = yarpc examples`

Reply:
Remove `@` from linting commands to make output readable in Travis logs

Reply:
because with `find` we ensure we are looking at th `py` files  in the entire project

Reply:
It does

Reply:
It does like this `mypy -m yarpc` so my question is, do you want to run all these linters only on the python files inside of `yarpc`?

Reply:
I don't understand why it doesn't work for you, does it work differently on different OS es?

I want to check these folders `yarpc examples`

Reply:
it does work as you specify `mypy yarpc` but that is only going to check the python files inside of the folder `yarpc` it is best using `ind` that will lint all the python files in the entire project, which is the best way and the proper way to do it.

Reply:
But all my files are in `yarpc` and `example` folders already, it's much easier to just give mypy (or any other tool) these folders instead of finding python files yourself. That's how it's usually done. I don't see a reason for doing this